crates/aptos/Cargo.toml

```
@@ -30,6 +30,7 @@ aptos-genesis = { workspace = true }
 aptos-github-client = { workspace = true }
 aptos-global-constants = { workspace = true }
 aptos-keygen = { workspace = true }
+aptos-ledger = "0.1.0"
```

re our chat irl, let's update this
```suggestion
aptos-ledger = { workspace = true }
```